### ISSUE TITLE
Training details for real image denoising

### ISSUE THREAD

**Issue by Shakarim94** [visinf/n3net#4]
BSDS, DIV2K and Waterloo datasets are used in the paper for real image denoising. I want to know how many patches were extracted for training. Did you use the same principle as in Gaussian denoising (512 80x80 patches from each image)?

What about the number of epochs and lr decay? I couldn't find those on the paper, they are given only for Gaussian denoising.

**Comment by tobiasploetz** [comment]
There are 50 epochs, with a decay as specified in [here](https://github.com/visinf/n3net/blob/458a6d6bb096973cf49ce219bc76c3fec1d0a812/src_denoising/experiment_pg.py#L171). The initial learning rate is 0.001. Each epoch sweeps 16 through the training set, cropping a 80x80 patch from each image.

I also tried a longer training (more iterations per epoch) but that did not result in better accuracy.
